### ISSUE TITLE
Add new recitation: Abdulrasheed Soufi 

### ISSUE THREAD

**Issue by weekmo** [quran/quran.com-api#312]

**Comment by ahmedre** [comment]
already present on quranicaudio and quran mobile apps so we have timing files if you need them Naveed

**Comment by ahmedre** [comment]
no need to close akhi, i was commenting telling @naveed-ahmad where he can get the files to add this to the website if people want it in sha' Allah.

**Comment by naveed-ahmad** [comment]
Can you send the timing files? Also, is there any tool that I can use to split audio into ayah by ayah using this timing file? @ahmedre

**Comment by weekmo** [comment]
Sorry, I thought you were telling me where to find it.

**Comment by ahmedre** [comment]
well, you can use our iOS or Android app and listen to the sheikh there, but if this is a request to add it to quran.com itself, i agree with you it'd be nice to do so 👍

**Comment by weekmo** [comment]
Yes, that is the reason of opening this issue [adding it to quran.com]

**Comment by ahmedre** [comment]
@naveed-ahmad in sqlite format - https://android.quran.com/data/databases/audio/abdurrashid_sufi.zip - just sura|ayah|timing, one row per ayah.
